packages/connect-web/src/iframe/index.ts

```diff
@@ -89,6 +89,9 @@ export const init = async (settings: ConnectSettings) => {
 
     instance.setAttribute('src', src);
     if (settings.webusb) {
+        console.log('webusb option is deprecated. use `transports: ["WebUsbTransport"] instead`');
```

Comment:
console.warn?

Reply:
fixed up in fixed up in https://github.com/trezor/trezor-suite/pull/7411/commits/96a0030d4d838643c0afd7a94831b93794170f64